1host.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	port: 3000,
+	modules: [
+		{
+			module: require("./index.js"),
+			errorHandler: false,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 3 tabs.](https://app.codacy.com/gh/onedotprojects/scratchCommentAuth/pullRequest?prid=7364619)
```suggestion
        errorHandler: false,
```